config/ROOT.scons

```diff
@@ -28,7 +28,7 @@ rootdict = Builder(generator = generate_action,
 rootenv = Environment(ENV={'PATH' : os.environ['PATH']})
 rootenv.ParseConfig("root-config --cflags --ldflags --libs")
 rootenv.Append(LIBS='-lSpectrum')
-rootenv.Append(LIBS=['-lFoam', '-lPyROOT'])
+rootenv.Append(LIBS=['-lFoam', '-lPyROOT','-lMinuit2'])
```

Comment:
Completely agree. I hate "smart" option setting. It is better to force the users to make their own choices. Anyway, the check is fairly simple to perform and Minuit2 is becoming a mandatory dependency, so a "check and fail" seems to be adequate.